python/meterpreter/ext_server_stdapi.py

```diff
@@ -1390,6 +1391,65 @@ def stdapi_sys_process_execute(request, response):
         response += tlv_pack(TLV_TYPE_CHANNEL_ID, channel_id)
     return ERROR_SUCCESS, response
 
+@register_function_if(has_windll)
+def stdapi_sys_process_get_info(request, response):
+    proc_h = packet_get_tlv(request, TLV_TYPE_HANDLE)['value']
```

Comment:
If you're trying to account for when the handle isn't present then this needs to use `get` instead because the [dictionary will be empty](https://github.com/rapid7/metasploit-payloads/blob/0f18eef4f812b2212ab6ad9f9c7f0e66ebb68a2f/python/meterpreter/meterpreter.py#L544).

```suggestion
    proc_h = packet_get_tlv(request, TLV_TYPE_HANDLE).get('value')
```